From: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Subject: disable building asterisk with -march=native
Bug-Debian: https://bugs.debian.org/842917
Forwarded: no

Don't add -march=native to standard CFLAGS. We need a generic binary.

Index: trunk/build_tools/cflags.xml
===================================================================
--- trunk.orig/build_tools/cflags.xml
+++ trunk/build_tools/cflags.xml
@@ -130,7 +130,7 @@
 		</member>
 		<member name="BUILD_NATIVE" displayname="Allow compiler to generate code optimized for the CPU on which the build is performed.">
 			<support_level>core</support_level>
-			<defaultenabled>yes</defaultenabled>
+			<defaultenabled>no</defaultenabled>
 			<depend>native_arch</depend>
 		</member>
 	</category>
